ui/src/components/admin/profile/profileTab.tsx

```
@@ -1,19 +1,54 @@
-          <p>Permissions: Coming soon!</p>
+
+        {/* Permissions Section */}
+        <div>
+          <h4 class="text-xl font-semibold text-gray-900 mb-4">
+            Your Permissions
+          </h4>
+          <ul class="space-y-3">
+            {auth.role() &&
+              roleDescriptions[auth.role() as Role].map(
+                (desc: string, index: number) => (
+                  <li class="flex items-center space-x-3 text-gray-700 hover:bg-gray-50 p-2 rounded-md transition-colors">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)